components/Init/src/camkes_vm_interfaces.h

```diff
@@ -13,6 +13,7 @@
 int ram_num_untypeds(void);
 int ram_get_untyped(int ut, uintptr_t *paddr, int *size_bits, seL4_CPtr *cap);
 
+unsigned long allocman_virtual_size(void);
```

Comment:
I think we should aim for having a h-file for each template's c-file. If I remember correctly we get this prototype then automatically added in the genrated `camkes.h`, which `main.c` includes already.

Reply:
Used the template header method.